### PR TITLE
Make sure 'calls.' gets added before call based stuff in metadata FQN…

### LINKED ISSUE #43: JsUtil.flatten loses 'calls' in workflow metadata
For calls in workflow metadata it's not prepending 'calls' on the flattening. It's not carrying the prefix around properly

### PR DESCRIPTION
**Pull request by geoffjentry**
…s. closes #43

### CODE REVIEW

File: src/main/resources/standardTestCases/threestep.test [hunk 1]
```diff
@@ -4,5 +4,13 @@ testFormat: workflowsuccess
 files {
   wdl: threestep/threestep.wdl
   inputs: threestep/threestep.inputs
-  metadata: threestep/threestep.metadata
+}
+
+metadata {
```
Comment (geoffjentry) [inline src/main/resources/standardTestCases/threestep.test:9]: I added this purely to have a second example

File: src/main/resources/standardTestCases/hello.test [hunk 2]
```diff
@@ -9,8 +9,8 @@ files {
 metadata {
   workflowName: hello
   status: Succeeded
-  "hello.hello.executionStatus": Done
+  "calls.hello.hello.executionStatus": Done
+  "calls.hello.hello.runtimeAttributes.docker": "ubuntu:latest"
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/hello.test:13]: One's a string, one isn't?

**Comment by cjllanwarne** [comment]
LGTM, but my 👍 only becomes active after you've reviewed #47...

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/48/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/hello.test (hunk 2 above)
Comment (geoffjentry) [inline src/main/resources/standardTestCases/hello.test:13]: hocon likes to have quotes around strings w/ punctuation

Comment (mcovarr) [inline src/main/resources/standardTestCases/hello.test:13]: hocon's cool like that

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/48/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**